lua/which-key/state.lua

```diff
@@ -235,7 +235,9 @@ function M.execute(state, key, node)
   end
   Util.debug("feedkeys", tostring(state.mode), keystr)
   local feed = vim.api.nvim_replace_termcodes(keystr, true, true, true)
-  vim.api.nvim_feedkeys(feed, "mit", false)
+  if Util.has_mapping(state.mode.mode, feed) then
+    vim.api.nvim_feedkeys(feed, "mit", false)
```

Comment:
Note that this will impact non-hydra mode as well... 

ie. 
![image](https://github.com/user-attachments/assets/388e69af-e00a-48e3-9a63-36c0d2d0041c)
in the past, `<space-j>` even though maps to nothing, it actually executes `j` (move down one line)

but after this change, `<space-j>` will be `no-op` instead

Reply:
this old commits broke the <C-R> registers ... 

so will be only checking the key_map when inside hydra_mode instead